app/frontend/lectures/edit/_media.html.erb

```diff
@@ -17,6 +17,7 @@
                                       teachable_type: 'Lecture'
                                     }),
                     class: 'btn btn-sm btn-secondary new-in-lecture',
+                    role: "button",
```

Comment:
@fosterfarrell9 Whenever you encounter a link that should be a button, please add this in your PRs, instead of using `getByRole("link")` in your Playwright tests.

For the difference between links and buttons, see [this](https://css-tricks.com/buttons-vs-links/). It's important for accessibility.